qa/suites/crimson-rados/seastore/basic/deploy/cephadm.yaml.disabled

```diff
@@ -0,0 +1,16 @@
+# no need to verify os + flavor + sha1
```

Comment:
The .disabled suffix here indicates that it's disabled, right?  Probably worth removing?

Reply:
done.